sources/packages/web/src/views/student/SINStatusBanner.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <Message :severity="severity">
```

Comment:
The <Message> can be directly used in CheckValidSINBanner.vue instead of being wrapped inside SINStatusBanner.vue.